docs/docs/plugins/02-dev-plugins.md

```diff
@@ -129,16 +132,16 @@ For instance, let's say your plugin adds a new `oracle` command to `ignite
 scaffold`, the `Manifest()` method will look like :
 
 ```go
-func (p) Manifest() (plugin.Manifest, error) {
-	return plugin.Manifest{
+func (p) Manifest(context.Context) (*plugin.Manifest, error) {
+	return &plugin.Manifest{
 		Name: "oracle",
-		Commands: []plugin.Command{
+		Commands: []*plugin.Command{
```

Comment:
It is a pointer because `plugin.Manifest`, which defines the `Commands` property, defines it as a pointer, but in general pointer requirements come from the way the proto generated code works.